src/stories/containers/TransparencyReport/components/TransparencyForecast/useTransparencyForecast.tsx

```diff
@@ -45,6 +50,22 @@ export const useTransparencyForecast = (currentMonth: DateTime, budgetStatements
     return _.sortBy(Object.values(dict), 'id');
   }, [currentMonth, budgetStatements]);
 
+  const getTransferRequestTargetBalanceColumn = useCallback((wallet: BudgetStatementWalletDto) => {
+    const targetWithTimeSpan: Pick<BudgetStatementWalletTransferRequestDto, 'target' | 'walletBalanceTimeStamp'> =
+      {} as BudgetStatementWalletTransferRequestDto;
+
+    const lastIndex = (wallet.budgetStatementTransferRequest ?? []).length - 1;
+    if (wallet.budgetStatementTransferRequest && wallet.budgetStatementTransferRequest.length > 0) {
+      targetWithTimeSpan.target = wallet.budgetStatementTransferRequest[lastIndex].target;
+      targetWithTimeSpan.walletBalanceTimeStamp =
+        wallet.budgetStatementTransferRequest[lastIndex]?.walletBalanceTimeStamp;
+    }
+
+    return targetWithTimeSpan;
+  }, []);
```

Comment:
It would be nice to move this function to `src\stories\containers\TransparencyReport\utils\budgetStatementsUtils.ts` in order to have the utility function in a separate file instead of declaring it inside the hook which requires to be wrapped with `useCallback`

Reply:
done